### ISSUE TITLE
Uncaught Error: ENOENT: no such file or directory

### ISSUE THREAD

**Issue by anilanar** [bradleyboy/tidy-tabs-atom#7]
[Enter steps to reproduce below:]

1. Have some file open in a tab (from the project)
2. Move the parent folder out of project.
3. Make a modification in another file that wasn't removed and save.

**Atom Version**: 1.0.11
**System**: Mac OS X 10.11
**Thrown From**: [tidy-tabs](https://github.com/bradleyboy/tidy-tabs-atom) package, v0.2.0

### Stack Trace

Uncaught Error: ENOENT: no such file or directory, stat '/Users/anilanar/Projects/xyz/index.js'

```
At /Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:809

Error: ENOENT: no such file or directory, stat '/Users/anilanar/Projects/xyz/index.js'
  at Error (native)
  at fs.statSync (fs.js:846:18)
  at Object.fs.statSync (ATOM_SHELL_ASAR.js:206:16)
  at Object.module.exports.TidyTabs.isCandidate (/Users/anilanar/.atom/packages/tidy-tabs/lib/tidy-tabs.coffee:69:15)
  at Object.<anonymous> (/Users/anilanar/.atom/packages/tidy-tabs/lib/tidy-tabs.coffee:87:39)
  at Object.module.exports.TidyTabs.getItemsToClose (/Users/anilanar/.atom/packages/tidy-tabs/lib/tidy-tabs.coffee:100:10)
  at Object.module.exports.TidyTabs.tidy (/Users/anilanar/.atom/packages/tidy-tabs/lib/tidy-tabs.coffee:48:5)
  at /Users/anilanar/.atom/packages/tidy-tabs/lib/tidy-tabs.coffee:40:18
  at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:82:11)
  at TextBuffer.module.exports.TextBuffer.saveAs (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:1038:20)
  at TextBuffer.module.exports.TextBuffer.save (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:1005:19)
  at TextEditor.module.exports.TextEditor.save (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:584:26)
  at Pane.module.exports.Pane.saveItem (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:547:18)
  at Pane.module.exports.Pane.saveActiveItem (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:530:19)
  at Workspace.module.exports.Workspace.saveActivePaneItem (/Applications/Atom.app/Contents/Resources/app.asar/src/workspace.js:609:35)
  at atom-workspace.atom.commands.add.core:save (/Applications/Atom.app/Contents/Resources/app.asar/src/workspace-element.js:293:30)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:245:29)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:524:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:347:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:184:20)

```

### Commands

```
     -0:32.7.0 core:move-right (atom-text-editor.editor.is-focused)
  3x -0:31.8.0 core:move-down (atom-text-editor.editor.is-focused)
     -0:30.6.0 editor:move-to-end-of-word (atom-text-editor.editor.is-focused)
     -0:30.2.0 editor:delete-to-beginning-of-word (atom-text-editor.editor.is-focused)
     -0:29.4.0 editor:move-to-end-of-screen-line (atom-text-editor.editor.is-focused.autocomplete-active)
  2x -0:29 editor:move-to-beginning-of-word (atom-text-editor.editor.is-focused)
     -0:26.2.0 core:move-right (atom-text-editor.editor.is-focused.autocomplete-active)
     -0:26 core:backspace (atom-text-editor.editor.is-focused)
     -0:24.9.0 editor:move-to-end-of-screen-line (atom-text-editor.editor.is-focused)
     -0:22.2.0 editor:newline (atom-text-editor.editor.is-focused)
     -0:18.8.0 core:move-down (atom-text-editor.editor.is-focused.autocomplete-active)
     -0:18.4.0 autocomplete-plus:confirm (atom-text-editor.editor.is-focused.autocomplete-active)
     -0:09.2.0 editor:move-to-end-of-screen-line (atom-text-editor.editor.is-focused)
     -0:08.3.0 core:move-down (atom-text-editor.editor.is-focused)
     -0:04.2.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:01.7.0 core:save (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "destroyEmptyPanes": false,
    "ignoredNames": [
      ".git",
      ".DS_Store",
      "gin-bin"
    ],
    "disabledPackages": [
      "language-javascript-jsx",
      "language-javascript-semantic"
    ],
    "themes": [
      "one-light-ui",
      "one-light-syntax"
    ]
  },
  "tidy-tabs": {
    "modifiedThreshold": 10,
    "runOnSave": true
  }
}
```

### Installed Packages

```coffee
# User
atom-jade, v0.3.0
coffee-compile, v0.18.0
go-plus, v3.5.1
highlight-selected, v0.10.1
jshint, v1.7.0
language-babel, v0.15.0
linter, v1.5.1
linter-coffeelint, v1.1.0
linter-jscs, v3.0.0
minimap, v4.13.3
minimap-highlight-selected, v4.3.0
project-manager, v2.5.1
rename-tabs, v0.1.1
tidy-tabs, v0.2.0
wordcount, v2.3.0

# Dev
No dev packages
```

**Comment by anilanar** [comment]
Duplicate of #4.

**Comment by bradleyboy** [comment]
Should be resolved in 0.3.0. Let me know if you still have problems.
